README.md

```diff
@@ -1,44 +1,36 @@
 <h1 align="center">ProtonVPN Linux GUI</h1>
 <p align="center">
-  <img src="https://i.imgur.com/rjMuf7p.png" alt="Logo"></img>
+  <img src="./protonvpn_linux_gui/resources/img/protonvpn_readme_logo.png" alt="ProtonVPN Logo"></img>
 </p>
 
 <p align="center">
-  <a href="https://github.com/calexandru2018/protonvpn-linux-gui/releases/latest">
-      <img alt="Build Status" src="https://img.shields.io/github/release/calexandru2018/protonvpn-linux-gui.svg?style=flat" />
+  <a href="https://github.com/ProtonVPN/protonvpn-linux-gui/releases/latest">
```

Comment:
As reported by rafficer separetly the official repo path is: https://github.com/ProtonVPN/linux-gui/
Can you amend it everywhere?

Reply:
Sure I'll fix that. It's just that when you updated the setup.py you left protonvpn-linux-gui, and I thought that it should be left like that since it is an "internal pointer" to this repo.

Reply:
Ah sure. But yeah better put resolved final paths, so that in case the redirection one day will stop working, everything stays correct.